package.json

```diff
@@ -108,23 +108,25 @@
 		"@nextcloud/browserslist-config": "^2.2.0",
 		"@nextcloud/eslint-config": "^8.0.0",
```

Comment:
> Update eslint-plugin-vue to latest version (9+) if you run into unused variable lint errors when using <script setup>.

For good measure uping eslint-config to 8.1 pulls in eslint-plugin-vue v9 instead of v8

Reply:
It is not release yet.
https://www.npmjs.com/package/@nextcloud/eslint-config

Reply:
Ah, nevermind then. We'll probably get the new version when it's out anyway.